### PR TITLE
Update UI for ONNX models

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,15 +14,18 @@ To use this module, follow these instructions to [add a module from the Viam Reg
...
+On the new component panel, deploy a model on to your machine by selecting an onnx model from the **Model** dropdown or by specifying a path to an existing model on your machine.
+
+If you prefer to configure your service using JSON, use the following attributes **attributes**:
 
 ```json
 {
   "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
-  "label_path": "/path/to/labels.txt"
+  "label_path": "/path/to/labels.txt",
+  "num_threads": 1
```
Comment (bhaney) [inline README.md]: num_threads is not actually a config attribute for this model  - onnx models can only run on 1 thread

**Review by bhaney — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (npentrel) [reply README.md]: Should it be removed from the app UI then?

Comment (npentrel) [reply README.md]: Because it does show up...

File: README.md [hunk 2]
```diff
@@ -37,6 +40,7 @@ The following attributes are available for the `viam-labs:mlmodel:onnx-cpu` ML m
 | ------- | ------ | ------------ | ----------- |
 | `model_path` | string | **Required** | The full path to the ONNX file. |
 | `label_path` | string | Optional | The full path to the names of the category labels. |
+| `num_threads` | int | Optional | The number of threads. |
```
Comment (npentrel) [inline README.md]:
```suggestion
| `num_threads` | int | Optional | The number of threads. Must be 1. |
| `package_reference` | string | Optional | The package of the model to deply. For example: "viam-soleng/face-detector-onnx". |

```

File: README.md [hunk 3]
```diff
@@ -14,15 +14,18 @@ To use this module, follow these instructions to [add a module from the Viam Reg
...
+
+If you prefer to configure your service using JSON, use the following attributes **attributes**:
 
 ```json
 {
   "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
-  "label_path": "/path/to/labels.txt"
+  "label_path": "/path/to/labels.txt",
+  "num_threads": 1
 }
```
Comment (npentrel) [inline README.md:30]:
```suggestion
  "package_reference": "viam-soleng/face-detector-onnx"
}
```

File: README.md [hunk 4]
```diff
@@ -14,15 +14,19 @@ To use this module, follow these instructions to [add a module from the Viam Reg
...
+On the new component panel, deploy a model on to your machine by selecting an onnx model from the **Model** dropdown or by specifying a path to an existing model on your machine.
+
+If you prefer to configure your service using JSON, use the following attributes **attributes**:
 
 ```json
 {
   "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
-  "label_path": "/path/to/labels.txt"
+  "label_path": "/path/to/labels.txt",
+  "num_threads": 1
```
Comment (npentrel) [inline README.md]:
```suggestion
  "num_threads": 1,
```

**Comment by npentrel** [comment]
@bhaney Are we ok to keep `num_threads` and say it must be 1? I think then we can merge this?

File: README.md [hunk 5]
```diff
@@ -37,6 +41,9 @@ The following attributes are available for the `viam-labs:mlmodel:onnx-cpu` ML m
 | ------- | ------ | ------------ | ----------- |
 | `model_path` | string | **Required** | The full path to the ONNX file. |
 | `label_path` | string | Optional | The full path to the names of the category labels. |
+| `num_threads` | int | Optional | The number of threads. Must be 1. |
+| `package_reference` | string | Optional | The package of the model to deply. For example: "viam-soleng/face-detector-onnx". |
```
Comment (bhaney) [inline README.md]: Do you know what the `package_reference` is? We don't use the field in the config of the module, the module would only know about the `model_path`, which should hold the path to where the model was deployed if the user used the registry to get their model

**Review by bhaney — COMMENTED** [review]
question about what `package_reference` is, I'm not familiar with it

**Comment by bhaney** [comment]
> @bhaney Are we ok to keep `num_threads` and say it must be 1? I think then we can merge this?

Yes, that works for me

File: README.md (hunk 5 above)
Comment (npentrel) [reply README.md]: right that's a problem in the UI as well then. I'll tag you in the ticket

Comment (npentrel) [inline README.md]:
```suggestion
| `package_reference` | string | Optional | The package of the model to deploy if the model was deployed through the registry. For example: "viam-soleng/face-detector-onnx". |
```

**Comment by npentrel** [comment]
@bhaney can you approve so we can merge this? Also ok if we want to remove package_reference for this but people may see it and get confused, so I think with the current explanation this should be ok

**Review by bhaney — APPROVED** [review]

**Pull request merged**